rules/mito_variants.smk

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+
+configfile: "config.yaml"
+
+samples = pd.read_table(config["run"]["samples"]).set_index("sample", drop=False)
+project = config["run"]["project"]
+
+rule mity_call:
+    input:
+        bam=[expand("recal/{family}_{sample}.bam".format(family=config["run"]["project"], sample=s)) for s in samples.index]
+    output:
+        protected("mitochondrial_variants/{family}.mity.vcf.gz")
+    params:
+        outdir="mitochondrial_variants/",
+        prefix="{family}",
+        tool=config["tools"]["mity"]
+    log:
+        "logs/mity/mity_call/{family}.mity_call.log"
+    wrapper:    
+        get_wrapper_path("mity/call")
+
+
+rule mito_vcfanno:
+    input:
+        "mitochondrial_variants/{family}.mity.vcf.gz"
+    output:
+        "mitochondrial_variants/vcfanno/{family}.mity.vcfanno.vcf"
+    log:
+        "logs/mity/vcfanno/{family}.mity.vcfanno.log"
+    threads: 10
+    resources:
+        mem_mb = 20000
+    params:
+       	conf ="~/crg2/vcfanno/mt.vcfanno.conf",
```

Comment:
Can you specify the vcfanno as a config parameter instead please? E.g. like [this](https://github.com/ccmbioinfo/crg2/blob/mity/config.yaml#L84).

Reply:
Done!